package es.udc.fic.sgsi_magerit;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class ThreatTest {
    @Test
    public void addition_isCorrect() throws Exception {
        //Thread.sleep(12741);
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect2() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect3() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect4() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect5() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect6() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect7() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect8() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect9() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect10() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect11() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect12() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect13() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect14() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect15() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect16() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect17() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect18() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect19() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect20() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect21() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect22() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect23() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect24() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect25() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect26() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect27() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect28() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect29() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect30() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect31() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect32() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect33() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect34() throws Exception {
        assertEquals(4, 2 + 2);
    }

    @Test
    public void addition_isCorrect35() throws Exception {
        assertEquals(4, 2 + 2);
    }
}
